lib/mix/lib/mix/compilers/elixir.ex

```diff
@@ -15,7 +17,7 @@ defmodule Mix.Compilers.Elixir do
     runtime_dispatches: [],
     external: [],
     warnings: []
-  ]
+  )
```

Comment:
just trying to understand this change here. is it right to say that everything after `:source` is matched as a keyword list for `defrecord/2`?